src/videojs-contrib-hls.js

```diff
@@ -524,6 +578,26 @@ class HlsHandler extends Component {
   }
 
   /**
+   * Initializes the quality levels and sets listeners to update them.
+   *
+   * @method setupQualityLevels_
+   * @private
+   */
+  setupQualityLevels_() {
+    let _player = videojs(this.tech_.options_.playerId);
```

Comment:
not really, was just following suit of already existing code https://github.com/videojs/videojs-contrib-hls/blob/master/src/videojs-contrib-hls.js#L287